jobs/job_calculate_precision.py

```diff
@@ -139,14 +141,21 @@ def check_fr(fr_list):
 
     return 0.5
 
-def check_rational(p, q):
+def check_rational(p, q, num):
     val = mpmath.mpf(p) / mpmath.mpf(q)
     if mpmath.almosteq(p, 0) or mpmath.almosteq(val, 0):
         logging.getLogger(LOGGER_NAME).debug(f'Checking rational that is too close to 0 p={p},q={q}')
         return 1
     
     if mpmath.pslq([val, 1], tol=mpmath.power(10, -100)):
         return 1
+    
+    integer_roots = math_utils.integer_roots(num)
+    for i in integer_roots:
+        if i > 0:
+            logging.getLogger(LOGGER_NAME).debug(f'numenator has positive integer root, num={num}, root={i}')
```

Comment:
numerator

Reply:
Also, try to run, if logging try to do it like: ...debug('text text %s', 'text')